src/stacks-hierarchy/Root_TicketAssistantStack/api.ts

```diff
@@ -0,0 +1,17 @@
+import {client} from '@atb/api';
+import {
+  RecommendedTicketResponse,
+  TicketAssistantData,
+} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/types';
+
+export async function getRecommendedTicket(
+  body: TicketAssistantData,
+): Promise<RecommendedTicketResponse> {
+  const url = 'ticket/v3/assistant';
+  try {
+    const response = await client.post<RecommendedTicketResponse>(url, body);
+    return response.data;
+  } catch (error) {
+    return Promise.reject(error);
+  }
```

Comment:
Catching an `error` to just throw it again is not necessary. Can then just skip the try/catch block.